test/unit/modules/utils/utils.spec.ts

```diff
@@ -212,7 +212,10 @@ describe('Utilities', () => {
       });
 
       test('should create a base url for registry', () => {
+        expect(combineBaseUrl("http", 'domain', '')).toEqual('http://domain');
+        expect(combineBaseUrl("http", 'domain', '/')).toEqual('http://domain');
         expect(combineBaseUrl("http", 'domain', '/prefix/')).toEqual('http://domain/prefix');
+        expect(combineBaseUrl("http", 'domain', '/prefix/deep')).toEqual('http://domain/prefix/deep');
         expect(combineBaseUrl("http", 'domain', 'only-prefix')).toEqual('only-prefix');
```

Comment:
```
expect(combineBaseUrl("http", 'domain', 'only-prefix')).toEqual('only-prefix');
```
🤔 ... this outcome is odd ... (not related with this PR but called my attention)

Reply:
In some cases `url_prefix` can be used as domain.
https://github.com/verdaccio/verdaccio/issues/622#issuecomment-371124490

Reply:
Thank u that makes totally sense .. then I would update the scenario with a real domain, a single string is a bit misleading.

Reply:
Will one of you create anew PR or issue for that?